src/agents/synthetic_worker.py

```
@@ -0,0 +1,74 @@
+"""
+Synthetic worker agents for orchestration CLI/load scenarios.
+"""
+
+import asyncio
+from typing import Any, Dict, Optional, Sequence, Set
```

Ruff (0.15.5)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)